.packit.yaml

```diff
@@ -487,7 +505,7 @@ jobs:
   targets:
     epel-8-x86_64:
       distros: [RHEL-8.6-rhui]
-  identifier: sanity-8to9-aws-e2e
+  identifier: sanity-8.6to9.0-aws-e2e
```

Comment:
@fernflower  @mm0ran  I guess it should be set 8.8to9.2 instead, as 8.6 to 9.0 is going to be dropped (I already wanted to drop it) from the enabled upgrade paths and we do not consider any other development related to this upgrade path, hence no testing.

The same about the `sanity-79to86-aws`, use rather 79to88. Thinking about that, 8.9 and 9.3 are outside. We want to drop these also, but I can keep them till Feb if wanted.

Reply:
Update: we are for now keeping also 8.6 & 9.0 upgrade paths until tests are updated.